samples/msal-node-samples/RedisTestApp/README.md

```diff
@@ -121,9 +121,9 @@ Session support in this sample is provided by the [express-session](https://www.
 
 ### Token caching
 
-MSAL Node has an in-memory cache by default. The demo app also features a [persistent cache plugin](./TestApp/App/utils/cachePlugin.js) in order to save the cache to disk. It illustrates a distributed caching pattern where MSAL's token cache is persisted via an external service (here, Redis) and only the currently served user's tokens (and other authentication artifacts) are loaded into MSAL's memory.
+MSAL Node has an in-memory cache by default. The demo app also features a [partition manager](./TestApp/App/utils/partitionManager.js). The manager illustrates a distributed caching pattern where MSAL's token cache is persisted via an external service (here, Redis) and only the currently served user's tokens (and other authentication artifacts) are loaded into MSAL's memory.
```

Comment:
I think this is a bit too detailed, but not sure of a proposal for something else at the moment. Will try to think of a suggestion.

Reply:
Shortened this bit with a link to the caching doc